src/pages/detailedpokemon/[id].tsx

```diff
@@ -76,6 +115,22 @@ export default function DetailedPokemon() {
         return total + current.base_stat;
     }, 0);
 
+    const engName = pokemonSpecies.names.find(
+        (pokename) => pokename.language.name === "en"
+    );
+    if (engName === undefined) {
```

Comment:
Not rendering the whole page is not a good default behavior. Please use a default value for the name instead.